### DISCUSSION TITLE
Why does $queryRaw throw an error here?

### DISCUSSION THREAD

**Discussion by janpio** [prisma/prisma#6620] [Q&A]
After creating my `PrismaClient`, I want to name the current connection with this raw query:

```
await prisma.$queryRaw`SET application_name to 'Prisma #${i}'`
```

Why does this query throw an error?

```
PrismaClientKnownRequestError2 [PrismaClientKnownRequestError]:
Invalid `prisma.queryRaw()` invocation:


  Your raw query had an incorrect number of parameters. Expected: `0`, actual: `1`.
    at cb (C:\Users\piotr\Documents\serverless-prisma\demo\node_modules\@prisma\client\runtime\index.js:34780:17)
    at processTicksAndRejections (node:internal/process/task_queues:94:5) {
  code: 'P1016',
  clientVersion: '2.21.2',
  meta: { expected: 0, actual: 1 }
}
```

Isn't that how it's supposed to be used?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
